### ISSUE TITLE
Add polling to VAN Bulk Import method

### ISSUE THREAD

**Issue by ChrisC** [move-coop/parsons#590]
To improve the user-friendliness of this method. For more context, see https://github.com/move-coop/parsons/pull/388#discussion_r488194754

**Comment by shaunagm** [comment]
Marking this as low priority because there's been no comments asking for it since Nov 2021. Can update if people disagree.

Might want to wait on this anyway, as if we're adding polling utility to APIConnector this VAN method would probably want to use it.
